docs/messageData.md

```diff
@@ -101,7 +110,7 @@ broadcast”).
         "id": {
             "type": "string",
             "contentEncoding": "base64",
-            "$comment": "Hash : SHA256(organizer||creation||name)"
+            "$comment": "Hash : HashLen(organizer, creation, name)"
```

Comment:
is HashLen defined somewhere?

Reply:
Yes, in the [`docs/protocol.md`](https://github.com/dedis/student_21_pop/blob/master/docs/protocol.md#concatenation-for-hashing) file